detox/src/devices/Device.test.js

```diff
@@ -547,11 +548,11 @@ describe('Device', () => {
 
   it(`should accept absolute path for binary`, async () => {
     const actualPath = await launchAndTestBinaryPath('absolutePath');
-    expect(actualPath).toEqual('/tmp/abcdef/123');
+    expect(actualPath).toEqual(process.platform === 'win32' ? 'C:\\Temp\\abcdef\\123' : '/tmp/abcdef/123');
   });
 
   it(`should accept relative path for binary`, async () => {
     const actualPath = await launchAndTestBinaryPath('relativePath');
-    expect(actualPath).toEqual(`${process.cwd()}/abcdef/123`);
+    expect(actualPath).toEqual(path.join(process.cwd(), 'abcdef/123'));
```

Comment:
Shouldn't this be split to `path.join(process.cwd(), 'abcdef', '123')` ?

Reply:
Path will normalise separators, and windows in fine with either except for binary paths (where they are ambiguous with switches)